packages/odyssey-react-mui/src/labs/SideNav/SideNav.tsx

```diff
@@ -160,7 +163,7 @@ const SideNavScrollableContainer = styled("div", {
 const SectionHeaderContainer = styled("li", {
   shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
 })(({ odysseyDesignTokens }: { odysseyDesignTokens: DesignTokens }) => ({
-  paddingBlock: odysseyDesignTokens.Spacing3,
+  paddingBlock: odysseyDesignTokens.Spacing1,
```

Comment:
This _probably_ doesn't need changed, I don't think

Reply:
I saw that it was adding lot of spacing around the header even in the compact layout, but the figma mock looks pretty tight. I can change it back if you think this would look better. here's the mock from Figma.
<img width="340" alt="image" src="https://github.com/user-attachments/assets/8ab5d77d-d26c-44fd-844d-ae138f175e4b">

Screenshot with the original Spacing3
<img width="273" alt="image" src="https://github.com/user-attachments/assets/6551f60a-a275-498a-9b75-f2125ca53d5e">

Screenshot from the above change.. Spacing1
<img width="274" alt="image" src="https://github.com/user-attachments/assets/005ee502-a667-4843-b7a4-c142eea8ba48">

Reply:
Ok, maybe they updated the design. You are right, this change looks more like the mocks. We can leave it. Thanks!